starter_code/src/movies.js

```diff
@@ -1,13 +1,2035 @@
+var movies = [
```

Comment:
You don't need to copy the variable in this file. 'movies.js' has access to 'data.js' because the second one is before in order in index.html.